web/templates/layout/header.html.eex

```diff
@@ -5,31 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="A package manager for the Erlang ecosystem">
-    <link rel="search" type="application/opensearchdescription+xml" title="Hex" href="/hexsearch.xml">
-    <link rel="stylesheet" href="<%= static_path(HexWeb.Endpoint, "/css/app.css") %>">
+
     <title><%= title(assigns) %></title>
+
+    <link rel="search" type="application/opensearchdescription+xml" title="Hex" href="/hexsearch.xml">
+    <link href='https://fonts.googleapis.com/css?family=Lato:400,300,700,900,100' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="<%= static_path(HexWeb.Endpoint, "/css/main.css") %>">
     <link rel="shortcut icon" href="/favicon.png">
+
   </head>
   <body>
-    <nav class="navbar navbar-default navbar-static-top" role="navigation">
+    <!--[if lt IE 10]>
+      <p class="browserupgrade">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+    <![endif]-->
```

Comment:
It's just a common way we use to tell people to upgrade their browsers in case they're using internet explorer. It never shows in the front-end unless they have IF below 10. I can remove, but it's a good way to tell people to upgrade their browser.